lib/maze/client/appium/bs_devices.rb

```diff
@@ -110,6 +111,10 @@ def create_hash
             add_android 'Samsung Galaxy S8', '7.0', hash                      # ANDROID_7_0_SAMSUNG_GALAXY_S8
 
             # Specific iOS devices
+            add_ios 'iPhone 15 Pro Max', '17.0', hash                         # IOS_17_0_IPHONE_15_PRO_MAX
```

Comment:
These two aren't valid option.  Have a look at https://www.browserstack.com/app-automate/capabilities for what BrowserStack have available - there are 5 different iOS phones with iOS 17.